NHSD.BuyingCatalogue.sln

```diff
@@ -105,154 +105,234 @@ Project("{00D1A9C2-B5F0-4AF3-8072-F6C62B433612}") = "NHSD.BuyingCatalogue.Databa
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NHSD.BuyingCatalogue.Database.Deployment", "src\NHSD.BuyingCatalogue.Database.Deployment\NHSD.BuyingCatalogue.Database.Deployment.csproj", "{20C95824-E0AD-488F-A903-BD0A9CABE847}"
 EndProject
+Project("{159641D6-6404-4A2A-AE62-294DE0FE8301}") = "NHSD.BuyingCatalogue.Database.PublishAll", "src\NHSD.BuyingCatalogue.Database.PublishAll\NHSD.BuyingCatalogue.Database.PublishAll.dtproj", "{7142A686-E318-4692-9C5E-DE74EC7873B9}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Development|Any CPU = Development|Any CPU
```

Comment:
Grateful if you could provide a brief comment just to outline the reason for the additional build configuration?

Reply:
The default configuration for SSIS is `Development`. Therefore, the development configuration was created automatically when I added the existing SSIS project. Given that it doesn't have any impact on the existing build setup I decided to leave as is.

In most cases there isn't any reason to have separate debug and release configurations for SSIS so (other than reducing the number of configurations) there would be no benefit in consolidating them.